how-to/customize-workspace/client/src/modules/auth/example/endpoint.ts

```diff
@@ -66,9 +66,10 @@ export class ExampleAuthEndpoint implements Endpoint<ExampleEndpointOptions> {
 
 		if (response.ok) {
 			const json = await response.json();
-			if (Array.isArray(json)) {
+			if (Array.isArray(json) || Array.isArray(json?.applications)) {
 				// returned apps
-				return this.applyCurrentUserToApps(json) as unknown;
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
There are a lot of casts, unknowns and lint disabling in these bits of code, any way we can make this a bit stricter ?